notify-server/Makefile

```
@@ -18,7 +18,8 @@ SRC_PATH = notify_server
 # Find the version of the wheel from package.json using a helper script. We
 # use python here so we can use the same version normalization that will be
 # used to create the wheel.
-wheel_file = dist/$(call poetry_python_get_wheelname,notify-server,notify_server,$(BUILD_NUMBER))
+wheel_file = dist/$(call python_get_wheelname,notify-server,notifyserver,$(BUILD_NUMBER))
```

I'm not really up to date with what `python_get_wheelname` expects, but is the Python package name `notify_server` (with the underscore), and if that's true, does that mean the call should be this?

```suggestion
wheel_file = dist/$(call python_get_wheelname,notify-server,notify_server,$(BUILD_NUMBER))
```

---

Just following the pattern of `robotserver`.